src/main/java/com/uid2/shared/store/parser/ServiceLinkParser.java

```diff
@@ -1,31 +1,28 @@
 package com.uid2.shared.store.parser;
 
-import com.uid2.shared.Utils;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 import com.uid2.shared.model.ServiceLink;
-import io.vertx.core.json.JsonArray;
-import io.vertx.core.json.JsonObject;
+import com.uid2.shared.util.Mapper;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.*;
+import java.util.stream.Collectors;
 
 public class ServiceLinkParser implements Parser<Map<String, ServiceLink>> {
 
+    private static final ObjectMapper OBJECT_MAPPER = Mapper.getInstance();
+
+    static {
+        OBJECT_MAPPER.enable(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE);
```

Comment:
Are we using default values for an enum?

Reply:
Good point! I just copied the object mapper pattern from clientkey and didn't look into the usage. I've looked at the documentation now, and we don't need it. Thanks for picking it up!

Reply:
I realised later that we do need it for roles. There is a default value 'UNKNOWN' already set up in Roles which allows the json creator to use that if the role parsed in is unknown.